### PR TITLE
Veronica

### PR DESCRIPTION
**Pull request by Viky397**
Finalized Readme. Waiting to add in Extension blurbs.

### CODE REVIEW

File: README.md
Comment (nmdickso) [inline README.md]: Packages which are either standard library or a dependency of another package should not be listed as requirements.

Comment (nmdickso) [reply README.md]: I can change that real quick

File: README.md
Comment (nmdickso) [inline README.md]: You don't navigate to the `scinet` folder, you go to wherever you cloned the repo, it's the `setup.py` file which is important, which will be wherever you clone it to, by default in `Phys490FinalProject`

**Comment by nmdickso** [comment]
I'm going to take a look and probably make a couple small changes to the copernicus blurb now as well, then that'll probably be it from me.

**Comment by Viky397** [comment]
Cool, thanks for the updates nolan.

**Review by nmdickso — DISMISSED** [review]

**Comment by nmdickso** [comment]
@keyblaede @PrinceOfPuppers Take a look and approve when you're happy with it please

**Comment by nmdickso** [comment]
I'll merge after we've all approved

**Review by nmdickso — APPROVED** [review]
Everyone's had a look, merging now.

**Pull request merged**
